static/app/views/monitors/utils.tsx

```diff
@@ -7,7 +7,7 @@ export function makeMonitorListQueryKey(
   organization: Organization,
   params: Record<string, any>
 ) {
-  const {query, project, environment, cursor, sort, asc} = params;
+  const {query, project, environment, owner, cursor, sort, asc} = params;
```

Comment:
hmm i know i wrote it like this way back, but for things like this should we just have destructured params straight into the query key? or is it good to explicitly declare each variable and put it in manually for transparency of what options we take?

Reply:
We want to explicitly declare, otherwise, other things in the query string will cause the API query to refetch.